app/validators/waste_exemptions_engine/legacy_manual_address_validator.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module WasteExemptionsEngine
+  class LegacyManualAddressValidator < ActiveModel::Validator
```

Comment:
This is here for convenience. Will be removed once this refactoring is applied to the other manual address forms.